proto/desmos/profiles/v1beta1/models_app_links.proto

```diff
@@ -0,0 +1,136 @@
+syntax = "proto3";
+package desmos.profiles.v1beta1;
+
+option go_package = "github.com/desmos-labs/desmos/x/profiles/types";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+// ApplicationLink contains the data of a link to a centralized application
+message ApplicationLink {
+  option (gogoproto.equal) = true;
+
+  // Data contains the details of this specific link
+  LinkData data = 1
+      [ (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"data\"" ];
```

Comment:
```suggestion
[
      (gogoproto.nullable) = false, 
      (gogoproto.moretags) = "yaml:\"data\"" 
];
```

Reply:
This can't be applied as the files are automatically formatted as you see using the `make proto-format` command

Reply:
Oh ok I thought they weren't formatted properly